content/hi/continuous_integration.md

```diff
@@ -0,0 +1,17 @@
+---
+title: निरंतर इंटीग्रेशन
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है 
+
+निरंतर इंटीग्रेशन , जिसे  ज्यादातर  सी.आई. बोला जाता है, यह कोड परिवर्तन को  जल्दी से जल्दी  एकीकृत करने का एक अभ्यास है। सी.आई. [सी.डी](/Continuous_Delivery/) पढ़ने के लिए एक ज़रूरी अवधारणा है। ज़्यादातर सी.आई. की प्रक्रिया तब शरू होती है जब कोड परिवर्तन  को स्रोत नियंत्रण सिस्टम (गित, सबवर्सन, और  मरक्यूरिअल) में कमिट किया जाता है और फिर वह ख़तम होता है एक परीक्षण किया हुए उत्पाद की तरह जो की तैयार है सी.डी सिस्टम में जाने के लिए।
```

Comment:
Let's have निरंतर एकीकरण to start with and continuous integration in parenthesis. As we are mentioning the abbreviation सी.आई. in the same sentence, it's ok to refer to that in remaining page.

Reply:
@Garima-Negi 
Do i have to add continuous integration in parenthesis next to सी.आई.?

Reply:
Done